app/presenters/bill-runs/two-part-tariff/review-licence.presenter.js

```diff
@@ -127,6 +128,18 @@ function _contactName (billingAccount) {
   return null
 }
 
+function _elementsInReview (licence) {
+  const hasReviewStatus = licence.reviewChargeVersions.some((chargeVersion) => {
```

Comment:
Had a chat with Alan about this. We think it should be possible to calculate if an element is in review and return a true/false at the licence level by updating the SQL query. I can pair with you on it if you like when you get back.